conda/recipes/libkvikio/variants_aarch64.yaml

```diff
@@ -0,0 +1,18 @@
+zip_keys:
+  - [c_compiler_version, cxx_compiler_version, cuda_version, should_use_cufile]
+
+c_compiler_version:
+  - 12
+  - 13
+
+cxx_compiler_version:
+  - 12
+  - 13
+
+cuda_version:
+  - 12.1 # The last version to not support cufile
+  - RAPIDS_CUDA_VERSION
+
+should_use_cufile:
+  - false
+  - true
```

Comment:
So we have this in `context` and this variant file. We should pick one

Personally would prefer `context` as there is a nice clear explanation and it is derived in a DRY way

Reply:
Thanks for catching this, I forgot some clean up here. We can use the context variable, but the condition there needs to be a bit more complex than it currently is. I'll fix that now.